js/main.js

```diff
@@ -227,16 +228,17 @@ function removeTags(str) {
 // this guy is going to populate the parent element in the dom with a container
 // containing buttons with event listeners
 // this is the prototype - looking to enhance this or move it into an object
-function recipeControls(parent, array) {
+function recipeControls(parent, object) {
```

Comment:
Tru to use more meaningful parameter/variable names. Here 'object' looks like it might contain recipe information, so a better name for the parameter might be `recipe`

Reply:
I will update this is a new branch.